packages/components/tag/src/tag-body.tsx

```diff
@@ -1,34 +1,32 @@
 import type { Theme } from '@emotion/react';
-import React, {
-  MouseEvent,
-  KeyboardEvent,
-  ReactNode,
-  ElementType,
-} from 'react';
+import type { TTagProps } from './tag';
+
+import React, { ReactNode, ElementType } from 'react';
 import styled from '@emotion/styled';
-import { Link } from 'react-router-dom';
-import { css, useTheme, SerializedStyles } from '@emotion/react';
+import { css, useTheme } from '@emotion/react';
 import {
   customProperties as vars,
   designTokens,
 } from '@commercetools-uikit/design-system';
 import Text from '@commercetools-uikit/text';
 
 export type TTagBodyProps = {
-  to?: string;
-  type: string;
-  onClick?: (
-    event: MouseEvent<HTMLDivElement> | KeyboardEvent<HTMLDivElement>
-  ) => void;
-  onRemove?: (
-    event: MouseEvent<HTMLDivElement> | KeyboardEvent<HTMLDivElement>
-  ) => void;
+  to?: TTagProps['to'];
+  as?: ElementType;
+  type?: string;
```

Comment:
hey why not `TTagProps['type']` here?

Reply:
The existing code has different types in `tag` and `tag-body`. 

But on second thought, having `'normal' | 'warning` in both places makes sense.